frontend/appflowy_flutter/packages/flowy_infra_ui/lib/style_widget/button.dart

```diff
@@ -1,12 +1,140 @@
 import 'dart:io';
 
-import 'package:flutter/material.dart';
-
 import 'package:flowy_infra/size.dart';
 import 'package:flowy_infra_ui/style_widget/hover.dart';
 import 'package:flowy_infra_ui/widget/flowy_tooltip.dart';
 import 'package:flowy_infra_ui/widget/ignore_parent_gesture.dart';
 import 'package:flowy_infra_ui/widget/spacing.dart';
+import 'package:flutter/material.dart';
+
+class FlowyIconTextButton extends StatelessWidget {
```

Comment:
Can't we reuse `FlowyTextButton` and simply provide a Row for the `text` (which takes a Widget) with the icon+text implementation?

Reply:
We need to customize the color of the text and icon on hover. I plan to migrate all the existing buttons to the new ones.